package.json

```diff
@@ -1401,22 +1404,12 @@
                             {
                                 "required": [
```

Comment:
Tasks in this screenshot: incorrect (both), incorrect (neither), correct (up), correct (down)
![image](https://user-images.githubusercontent.com/36966225/127540152-a1d085a4-bd88-420c-a721-d43ed158d43b.png)